library/common/api/external.cc

```diff
@@ -0,0 +1,25 @@
+#include "external.h"
+
+#include <unordered_map>
+
+#include "common/common/assert.h"
+
+namespace Envoy {
+namespace Api {
+namespace External {
```

Comment:
```suggestion
namespace PlatformExtension {
```

I think this meshes well with the rest of the PR, because it tells us that these function signatures allow us to register _any_ platform extension api?

Reply:
Sure - it may be obvious that I waffled a bit on naming and "external" was just a possibility I considered, too.

Reply:
Now that I've updated everything else to standardize on `PlatformBridge`, I actually think this should remain `External`. _Any_ API external to the library could be registered here, but everything labeled `PlatformBridge` relates specifically to bridging up to a platform runtime environment.